webpack/assets/javascripts/react_app/components/AutoCompleteSearch/helpers.js

```diff
@@ -1,33 +1,20 @@
 import React from 'react';
 import { Menu, MenuItem } from 'react-bootstrap-typeahead';
+import { groupBy } from 'lodash';
```

Comment:
please use:
```js
import groupBy from 'lodash/groupby'
```
this ensure we only load the parts we need from lodash vs all of it.